cypress/integration/app_spec.ts

```diff
@@ -1,70 +1,121 @@
 /* eslint-disable jest/valid-expect, jest/expect-expect */
 function setupServerMocks() {
-  cy.intercept('GET', 'https://freesound.org/apiv2/sounds/462808', {fixture:  'sound.json'})
-  cy.intercept('GET', '/people/qfox123/packs/26158', {fixture: 'sound.json'})
-  cy.intercept('GET', 'https://freesound.org/apiv2/packs/26158/', {fixture:'pack.json'})
-  cy.intercept('GET', 'https://freesound.org/apiv2/packs/26158/sounds/?fields=id%2Cname%2Cduration%2Cnum_downloads%2Cusername%2Cnum_ratings', {fixture:'pack-sounds.json'})
-  cy.intercept('GET', 'https://freesound.org/apiv2/sounds/462808/similar/?fields=id%2Cname%2Cduration%2Cnum_downloads%2Cusername%2Cnum_ratings', {fixture:'similar-sounds.json'})
-  cy.intercept('GET', 'https://freesound.org/data/previews/462/462808_8386274-lq.mp3', {fixture:'sound.mp3'})
-  cy.intercept('GET', 'https://freesound.org/apiv2/search/text/*', {fixture:'search-results.json'})
+  cy.intercept("GET", "https://freesound.org/apiv2/sounds/462808", {
+    fixture: "sound.json",
+  });
+  cy.intercept("GET", "/people/qfox123/packs/26158", { fixture: "sound.json" });
+  cy.intercept("GET", "https://freesound.org/apiv2/packs/26158/", {
+    fixture: "pack.json",
+  });
+  cy.intercept(
+    "GET",
+    "https://freesound.org/apiv2/packs/26158/sounds/?fields=id%2Cname%2Cduration%2Cnum_downloads%2Cusername%2Cnum_ratings",
+    { fixture: "pack-sounds.json" }
+  );
+  cy.intercept(
+    "GET",
+    "https://freesound.org/apiv2/sounds/462808/similar/?fields=id%2Cname%2Cduration%2Cnum_downloads%2Cusername%2Cnum_ratings",
+    { fixture: "similar-sounds.json" }
+  );
+  cy.intercept(
+    "GET",
+    "https://freesound.org/data/previews/462/462808_8386274-lq.mp3",
+    { fixture: "sound.mp3" }
+  );
+  cy.intercept(
+    "GET",
+    "https://freesound.org/apiv2/search/text/?fields=id%2Cname%2Cpreviews%2Cduration%2Cnum_downloads%2Cusername%2Cnum_ratings&page_size=10&query=hello",
+    {
+      fixture: "search-results.json",
+    }
+  );
+
+  cy.intercept(
+    "GET",
+    "https://freesound.org/apiv2/search/text/?fields=id%2Cname%2Cpreviews%2Cduration%2Cnum_downloads%2Cusername%2Cnum_ratings&page_size=10&page=2&query=hello",
+    {
+      fixture: "search-results-page-2.json",
+    }
+  );
 }
 
 describe("Home Page", () => {
   beforeEach(() => {
-    setupServerMocks()
-  })
+    setupServerMocks();
+  });
 
   it("should load the home page", () => {
     cy.visit("/");
-    cy
-      .get(`[data-e2e-id='SoundList']`)
-      .should('be.visible')
+    cy.get(`[data-e2e-id='SoundList']`).should("be.visible");
+  });
+});
+
+describe("Pagination element functionality", () => {
+  beforeEach(() => {
+    setupServerMocks();
+  });
+
+  it("checks if next page button shows next page of results", () => {
```

Comment:
testing convention is to start test name with `should`. ex:

```
it('should foo', () => {});
```

Reply:
I renamed the test to say `it("should show the second page of search results", () => {` in 9707d1b. I'm actually not sure if we can improve the name further, still, so I'll leave this conversation open.